src/usr/local/www/system_usermanager_settings.php

```diff
@@ -102,6 +102,8 @@
 
 $pconfig['backend'] = &$config['system']['webgui']['backend'];
 
+$pconfig['auth_refresh_time'] = &$config['system']['webgui']['auth_refresh_time'];
```

Comment:
Since this was my first foray into editing the GUI I played it safe and copied the way the rest of the values had been set. Updating all instances of assign by reference in this file for consistency.